templates/cisco_asa_show_version.template

```diff
@@ -19,9 +20,10 @@ Value LAST_MOD (.+)
 Start
   ^.*Software\sVersion\s${VERSION}
   ^Device.+\s${DEVICE_MGR_VERSION}
+  ^Compiled on \w+\s+${COMPILE_DATE}.*
   ^System image file.+"${IMAGE}"
   ^${HOSTNAME} up ${UPTIME}
-  ^Hardware:\s+${HARDWARE},
+  ^Hardware:\s+${HARDWARE}
```

Comment:
Can you keep the hardware line you deleted above the one you changed for hardware? This allows us to capture both instances without adding the extra comma.

Reply:
The checks fail with removing the comma on the raw2 file if I recall properly. I can retest again though.

Reply:
I think that the no comma is needed for the use case found on the raw2 file. Take a peak at the couple of build attempts on #470. @FragmentedPacket